### PR TITLE
Add wiring instructions to the quick start guide

### PR DESCRIPTION
**Pull request by thePunderWoman**
This adds some instructions on how to wire the belay for users, like myself, who may be new to wiring, controller boards, and klipper.

Note:
I took a photo of the Octopus v1.1 pointing at the port I used, but I wasn't sure if it would help or hurt. Let me know what you think and I can add that in with a "This photo illustrates an example of the endstop port on a controller board, in this case an Octopus v1.1".

### CODE REVIEW

File: docs/Quick_Start.md [hunk 1]
```diff
@@ -99,11 +99,28 @@ Insert the exit bowden tube, then add a collet clip:
 
 ![](images/add_exit_tube_and_clip.png?raw=true)
 
-Solder 2 wires to the microswitch and secure them to the sensor housing with a
-zip tie as shown below:
+## Wiring
+
+The microswitch has 3 solder points, but we're only going to be using 2 of them. Specifically, we'll be using the two solder points on the left and right sides of the housing. You can ignore the middle one.
+
+The wires will not be used to carry voltage. So you don't have to worry about using a specific guage of wire.
```
Comment (rsghosh) [inline docs/Quick_Start.md]:
```suggestion
The wires will not be used to carry any substantial current, so you don't have to worry about using a specific gauge of wire.
```

File: docs/Quick_Start.md [hunk 2]
```diff
@@ -99,11 +99,28 @@ Insert the exit bowden tube, then add a collet clip:
...
+Solder 2 wires to the microswitch as shown below:
+
+![](images/wiring_example_2.jpg?raw=true)
+
+Make sure you secure them to the sensor housing with a
+zip tie as shown below. This will keep any stresses from moving the belay around off the solder points.
 
 ![](images/wiring_example.jpg?raw=true)
 
+Next, you'll need to identify a port on your controller board you want to use. An endstop port is the easiest option, but you can use whatever is open. Make sure you note the sensor pin id of this port, as you'll need it when we get to setting this up in klipper later. In this example, we'll be using an endstop port. This is a 3-pin port, and connects using a JST-XH connector.
```
Comment (rsghosh) [inline docs/Quick_Start.md]:
```suggestion
Next, you'll need to identify a port on your controller board to use. An endstop port is the easiest option, but you can use almost any available port with an input signal pin and a ground pin. Make sure you note the microcontroller pin name for the signal pin of this port, as you'll need it when we get to setting up the belay in Klipper later. In this example, we'll be using an endstop port. This is a 3-pin port, and connects using a JST-XH connector.
```

File: docs/Quick_Start.md [hunk 3]
```diff
@@ -99,11 +99,28 @@ Insert the exit bowden tube, then add a collet clip:
...
+
+The port's pins are 3.3v, ground, and signal. The Belay only needs to connect to the ground and signal pins. You'll need to crimp the wires and then make sure you insert them in the connector aligned with the proper pins. It does not matter which wire is connected to ground or signal.
+![](images/wire_crimp_example.jpg?raw=true)
+
+That's all it takes to wire the belay. Whenever you're ready, you can run the wire and plug it in to your printer's controller board. Remember that sensor pin id for the next steps.
```
Comment (rsghosh) [inline docs/Quick_Start.md]:
```suggestion
That's all it takes to wire the belay. Whenever you're ready, you can run the wire and plug it in to your printer's controller board. Remember that sensor pin name for the next steps.
```

**Review by rsghosh — CHANGES_REQUESTED** [review]
Thanks! This looks great overall, I just left a few very minor suggested changes.

I think it is fine without the photo of the Octopus, since the endstop port location will depend on the board being used and should be explained by the board's documentation.

File: docs/Quick_Start.md (hunk 1 above)
Comment (thePunderWoman) [reply docs/Quick_Start.md]: Updated!

File: docs/Quick_Start.md (hunk 2 above)
Comment (thePunderWoman) [reply docs/Quick_Start.md]: Updated

File: docs/Quick_Start.md (hunk 3 above)
Comment (thePunderWoman) [reply docs/Quick_Start.md]: Aaaaand updated.

**Pull request merged**